src/InvertedIndices.jl

```diff
@@ -27,6 +27,18 @@ inverted index will similarly span multiple dimensions.
 """
 InvertedIndex, Not
 
+# A wrapper for `InvertedIndex` if it is used in broadcasting context.
+# Since `InvertedIndex` does not have a reference to the collection it applies to
+# it is impossible to define its axes properly.
+# Therefore it is a responsiblitie of the caller to properly resolve the handling
+# of the broadcast result. Intended example use in DataFrames.jl:
+# `combine(df, Not(:x) .=> fun)`
+# that would apply transformation `fun` to all columns of `df` except `:x`.
```

Comment:
I'd suggest not mentioning DataFrames here, especially given that this example requires familiarity with our special APIs to understand what it means.
```suggestion
# of the broadcast result.
```